haxepunk/Screen.hx

```diff
@@ -50,10 +48,6 @@ class Screen
 		{
```

Comment:
Now that it's only called once we could probably remove the init method and just make this initialization part of the constructor.